fdbserver/workloads/GetMappedRange.actor.cpp

```diff
@@ -433,6 +486,12 @@ struct GetMappedRangeWorkload : ApiWorkload {
 		} else if (r < 0.75) {
 			matchIndex = MATCH_INDEX_UNMATCHED_ONLY;
 		}
+		const double r2 = deterministicRandom()->random01();
+		if (r2 < 0.5) {
+			(const_cast<ServerKnobs*> SERVER_KNOBS)->STRICTLY_ENFORCE_BYTE_LIMIT = true;
+		} else {
+			(const_cast<ServerKnobs*> SERVER_KNOBS)->STRICTLY_ENFORCE_BYTE_LIMIT = false;
+		}
```

Comment:
```suggestion
		(const_cast<ServerKnobs*> SERVER_KNOBS)->STRICTLY_ENFORCE_BYTE_LIMIT = deterministicRandom()->coinFlip();
```

Also, maybe it's a good idea to save the knob value and restore it at the end of the test, in case other workloads running after this one don't encounter surprises.